changedetectionio/__init__.py

```diff
@@ -184,11 +189,13 @@ def hide_referrer(response):
 
     if ssl_mode:
         # @todo finalise SSL config, but this should get you in the right direction if you need it.
-        eventlet.wsgi.server(eventlet.wrap_ssl(eventlet.listen((host, port), s_type),
-                                               certfile='cert.pem',
-                                               keyfile='privkey.pem',
-                                               server_side=True), app)
-
+        ssl_args = {
+                'certfile' : 'cert.pem',
+                'keyfile' : 'privkey.pem',
+                'server_side' : True,
+               }
     else:
-        eventlet.wsgi.server(eventlet.listen((host, int(port)), s_type), app)
-
+        ssl_args = {}
+    sock = WSGIServer.get_listener((host, port), family=s_type)
+    cd_server = WSGIServer(sock, app, **ssl_args)
```

Comment:
I've seen this around on the internet, but i cant find any accurate documentation for it...

```suggestion
    # changedetection.io is actually a singled threaded application and stores all
    # data in the datastore dictionary, we will upgrade this in the future somehow.

    cd_server = WSGIServer(sock, app, **ssl_args, environ={'wsgi.multithread': False,'wsgi.multiprocess': False,})

```

Reply:
Is this for the nginx and so on?

Reply:
It has nothing todo with nginx, it should only allow 1 thread to run at any time.

nginx is only forwarding the connections and not starting a new application (the app is not CGI)

Reply:
Thank you!